seqr/utils/elasticsearch/es_search.py

```diff
@@ -434,7 +436,7 @@ def _process_single_search_response(self, parsed_response, page=1, num_results=1
         return variant_results[:num_results]
 
     def _execute_multi_search(self, **kwargs):
-        indices = self._index_searches.keys() or self._indices
+        indices = sorted(list(self._index_searches.keys()) or self._indices, reverse = True)
```

Comment:
given `self._indices` is already sorted, I think this would be neater as `sorted(list(self._index_searches.keys()), reverse = True)  or self._indices`

Reply:
True. Change it to `sorted(self._index_searches.keys(), reverse = True) or self._indices`.